src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -177,6 +194,16 @@ public UserMetadata getUserMetadata() {
   }
 
   /**
+   * Returns additional custom claims set on this user.
+   *
+   * @return a non-empty Map of custom claims or null.
+   */
+  @Nullable
+  public Map<String,Object> getCustomClaims() {
+    return customClaims;
```

Comment:
I think we should return an immutable copy here.

Also - can we return an empty map if there are no custom claims?

Reply:
Setting to empty map when there are no claims. This always get set to ImmutableMap in the constructor.